.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: Tests
```

Comment:
Is this file migrating the way tests are running to use docker instead of the php image I was using before ?